lib/components/bm-confirm-location-on-map.js

```diff
@@ -0,0 +1,148 @@
+/* @flow */
+'use strict'
+
+const geolocation = require('@blinkmobile/geolocation')
+
+/* :: import type {
+  AngularCallbackOptions, Coordinates, NgModelController
+} from '../../types.js' */
+
+const mod = require('../module.js')
+const utils = require('../utils.js')
+
+class BmConfirmLocationOnMapController {
+  /* :: static $inject : string[] */
+
+  // internal use
+
+  /* :: $digest: () => void */
+  /* :: coords: ?Coordinates */
+  /* :: isEditing: boolean */
+
+  // public attributes (after casts / checks)
+
+  /* :: ngDisabled: boolean */
+  /* :: ngModel: NgModelController */
+  /* :: ngReadonly: boolean */
+
+  constructor ($scope /* : Object */) {
+    this.$digest = $scope.$digest.bind($scope)
+    this.coords = null
+    this.isEditing = false
+  }
+
+  $onInit () {}
+
+  $onChanges () {
```

Comment:
I certainly could use `$onChanges(changes)` to avoid doing type-checks and conversions for bindings that haven't changed. Probably worth doing, just from a performance good-practices perspective.